rest-api/src/main/java/com/cloudbees/workflow/rest/external/ChunkVisitor.java

```diff
@@ -64,9 +65,16 @@ public static AtomFlowNodeExt makeAtomNode(@Nonnull WorkflowRun run, @CheckForNu
 
     @Override
     /** Do the final computations to materialize the stage */
+    @SuppressFBWarnings(value = "RCN_REDUNDANT_NULLCHECK_OF_NONNULL_VALUE", justification = "We can actually get nulls")
```

Comment:
FB wasn't inspecting deeply enough -- there was a case it misses.   This is the tradeoff for expecting fast testing.